specification/specs/aggregations/bucket/date_range/DateRangeExpression.ts

```diff
@@ -19,9 +19,5 @@
 
 class DateRangeExpression {
   from?: DateMath | float
-  from_as_string?: string
-  to_as_string?: string
-  key?: string
   to?: DateMath | float
-  doc_count?: long
```

Comment:
Those should not be removed :)

https://www.elastic.co/guide/en/elasticsearch/reference/7.12/search-aggregations-bucket-daterange-aggregation.html

Reply:
Yes they can be removed. `DateRangeExpression` is only used in `DateRangeAggregation` (the query part) that doesn't have these fields.

The `*_as_string` fields exist in the aggregation result buckets. They are currently not explicitely defined in `Aggregate` and are considered as valid because of how buckets are defined: `class KeyedBucket<TKey>
  implements AdditionalProperties<AggregateName, Aggregate>`.

I have some pending changes on aggregations that explicitely adds those `*_as_tring` fields (among other things) and will be part of the follow-up PR mentioned in the description.